internal/dataplane/parser/translate_utils.go

```diff
@@ -208,3 +155,76 @@ func (p *Parser) generateKongServiceFromBackendRef(
 
 	return service, nil
 }
+
+// func (p *Parser) generateKongServiceFromBackendRefV1Alpha2(
```

Comment:
What is this function used for? Do we need to keep it in the repo and commented? If so, I think we need to add some comments to clarify why we want to keep it.

Reply:
Nope, we don't need it. Thanks.

Reply:
Removed